onnxruntime/hosting/executor.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <google/protobuf/stubs/status.h>
+
+#include "environment.h"
+#include "predict.pb.h"
+
+namespace onnxruntime {
+namespace hosting {
+
+class Executor {
+ public:
+  Executor(HostingEnvironment* hosting_env, std::string request_id) : env_(hosting_env),
+                                                                                      request_id_(std::move(request_id)),
+                                                                                      using_raw_data_(true) {}
+
+  // Prediction method
+  google::protobuf::util::Status Predict(const std::string& model_name,
+                                         const std::string& model_version,
+                                         onnxruntime::hosting::PredictRequest& request,
+                                         /* out */ onnxruntime::hosting::PredictResponse& response);
+
+ private:
+  HostingEnvironment* env_;
```

Comment:
Can this be const?

Reply:
Unfortunately no because `GetLogger(string requestId)` cannot be const